src/test-run/commands/base.d.ts

```diff
@@ -6,6 +6,9 @@ export class CommandBase {
     public type: string;
     [key: string]: unknown;
     public _getAssignableProperties(): { name: string }[];
+    public _getAllAssignableProperties(): { name: string }[];
+    public _getNonReportedProperties(): string[];
+    public _getReportedProperties(): string[];
```

Comment:
Mark the new methods as public (and the `_getAssignableProperties` method too). These methods are public due to they are used in the outer code.